src/main/kotlin/gg/essential/gradle/multiversion/apivalidation/ExtractApiFile.kt

```diff
@@ -20,7 +21,7 @@ abstract class ExtractApiFile : DefaultTask() {
 
     @TaskAction
     fun merge() {
-        val parser = Parser(project.parent!!.childProjects.keys)
+        val parser = Parser(project.parent!!.multiversionChildProjects.keys)
```

Comment:
Similar concern here. Unlike the merge task, it's perfectly fine to extract the mappings for one of the projects only. So if I do `:1.12.2-forge:build` with on-demand configuration enabled, then some of the projects in here might not be configured.
Though it probably doesn't even matter here because we only really care about the project for which we extract the files, and that one's guaranteed to be configured otherwise we wouldn't be here.
Just need to make sure trying to access the other subprojects won't result in a "too late to configure this project now" error.

(the merge task config has a `evaluationDependsOnChildren()`, so it'll probably be fine)